its/src/test/java/com/sonar/it/csharp/MultipleProjectsTest.java

```diff
@@ -85,11 +89,16 @@ protected void before() throws Throwable {
 
           TestUtils.runMSBuild(ORCHESTRATOR, projectDir, "/t:Rebuild");
 
-          ORCHESTRATOR.executeBuild(TestUtils.createEndStep(projectDir));
+          buildResult = ORCHESTRATOR.executeBuild(TestUtils.createEndStep(projectDir));
         }
       });
   }
 
+  @Test
+  public void projectTypesInfoIsLogged() {
+    assertThat(buildResult.getLogs()).contains("Found 4 MSBuild projects. 2 MAIN project(s). 2 TEST project(s).");
```

Comment:
We have `pluralize(..)` duplicated in `ProtobufDataImporter` and `RoslynDataImporter`. It would be useful here as well so it would be worth extracting it to a reusable method.